cryptofeed/exchanges/bybit.py

```diff
@@ -193,6 +192,13 @@ async def subscribe(self, connection: AsyncConnection):
                             "args": [f"{chan}.{pair}"] if self.exchange_channel_to_std(chan) != CANDLES else [f"{chan if sym.quote == 'USD' else 'candle'}.{self.candle_interval_map[self.candle_interval]}.{pair}"]
                         }
                     ))
+            else:
+                await connection.write(json.dumps(
```

Comment:
thanks, I think this was lost in the change over to the new way of handling connections